### PR TITLE
Auto k8s worker disk cleanup

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Sets up Auto k8s worker disk cleanup

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- n/a

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: infra/ansible/roles/k8s/templates/disk_cleanup.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+systemctl stop docker.service
+systemctl stop docker.socket
+rm -r /var/lib/docker/*
+rm -r /run/log/*
+systemctl start docker.service
+systemctl start docker.socket
```
Comment (123MwanjeMike) [inline infra/ansible/roles/k8s/templates/disk_cleanup.sh.j2:7]: [disk cleanup script](https://github.com/airqo-platform/AirQo-api/pull/1521/commits/59b2696f798acfb2e15ccc44e0f062337480469a)

File: infra/ansible/cleanup_k8s_workers_disks.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        dest: /root/disk_cleanup.sh
+        mode: a+x
+
+    - name: Create cronjob for running the cleanup script
+      cron:
+        name: Clean up disk space
+        weekday: 6
+        hour: "{{ groups['k8s_workers'].index(inventory_hostname) }}"
+        minute: 0
+        job: sudo /bin/bash /root/disk_cleanup.sh
```
Comment (123MwanjeMike) [inline infra/ansible/cleanup_k8s_workers_disks.yml:16]: [playbook for disk cleanup](https://github.com/airqo-platform/AirQo-api/pull/1521/commits/d3dca2459980779aca9ddf3c26e0c796bd5fee65)

**Review by Baalmart — APPROVED** [review]
Thanks @123MwanjeMike

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**